### PR TITLE
Central moment interface and logic

### PR DESCRIPTION
**Pull request by aml5600**
* Computes `n` central moments of the function `g` using the Koopman `expectation` interface

Work In Progress

### CODE REVIEW

**Comment by aml5600** [comment]
@agerlach I will address your comments from Slack, wanted to get something up before the weekend =]

File: src/koopman.jl [hunk 1]
```diff
@@ -350,4 +350,39 @@ function expectation(g::Function, S::Function, u0, p, expalg::MonteCarlo, args..
...
+args and kwargs.
+
+Return: n - 1 (or 0) sized array of the 2 to n central moments
+
+Note: the zeroth and first central moment's are always 1 and 0, respectively.
+      As such, they are excluded in the return value
+
+TODO: - add support for vector-valued g functions, currently assumes scalar 
+      return values.
+      - add tests
```
Comment (aml5600) [inline src/koopman.jl:369]: FYI @agerlach

**Comment by aml5600** [comment]
@agerlach added the first moment (always 0) back to return value. Something interesting is that given the quadrature method, the computed values of `centralmoment(3, ... )` do not match those computed by `centralmoment(4, ... )` because of different sampled points.

**Comment by agerlach** [comment]
> the computed values of centralmoment(3, ... ) do not match those computed by centralmoment(4, ... ) because of different sampled points.

As in somethings wrong, or the 4 moment is more accurate because more sampled point. Note: that the reltol is set to 1e-2 by default. kwargs are `ireltol` and `iabstol`. 

Have you tested the function vs. a truth? E.g. MC or use an ODE w/ no dynamics and then use `Distributions.jl'

**Comment by aml5600** [comment]
Different digits as far as the "precision" goes, used the values from the code snippet you gave me on Slack

**Pull request merged**